frontend/src/components/Header.js

```diff
@@ -28,43 +28,45 @@ export default function Header({ children, user }) {
 
 	return (
 		<>
-			<Box style={{ width: "100%" }} sx={{ flexGrow: 1 }}>
-				<AppBar position="static" style={{ backgroundColor: "rgba(0,0,0,0)", boxShadow: "0 0 0 0" }}>
-					<Toolbar style={{ overflowX: "none", display: "flex", justifyContent: "space-between" }}>
-						<div style={{ display: "inherit" }}>
-							<Link to="/" style={{ textDecoration: "none", display: "inherit", userSelect: "none" }}>
-								<img src={NftLogo} alt="Logo" style={{ width: "63px", height: "63px", marginRight: "7px" }} />
-								<Typography style={{ fontSize: "17px", fontFamily: "Josefin Sans", lineHeight: "96%", marginTop: "19px", opacity: "86%", color: "white", whiteSpace: "nowrap" }}>
-									NFT<br/>the world!
-								</Typography>
-							</Link>
-							<div style={{ marginTop: "29px", marginLeft: "14px", display: "inherit" }}>
-								<div style={{ width: "15px", height: "0", transform: "rotate(270deg)", border: "1px solid #8d8d8d", color: "#8d8d8d", backgroundColor: "#8d8d8d", float: "left", marginRight: "14px", marginTop: "11px" }}/>
-								<Link to="/faq"  style={{ textDecoration: "none", color: "white" }}>
-									<Typography style={{ fontFamily: "Noto Sans", fontSize: "16px", fontStyle: "normal", fontWeight: "700", lineWeight: "22px", letterSpacing: "0em", textAlign: "left" }}>
-										FAQ
+			<header>
```

Comment:
The only change is this inserted header-tag, right?

Reply:
Pretty much.